source/reference/json/file-options-online-archive.txt

```diff
@@ -0,0 +1,99 @@
+.. _atlas-cli-file-options-online-archive-file:
+
+========================================
+Online Archive Configuration File
+========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use an Online Archive configuration file to specify the required 
+settings for creating and updating an Online Archive. This archive stores 
+data from one cluster within one project. To use this resource, the 
+requesting API Key must have the Project Data Access Admin role.
+
+Use the following resources to:
+
+- Learn the required settings you can specify in the Online Archive 
+  configuration file.
+
+- View and copy sample configuration files.
+
+Required Online Archive Create and Update Settings
+--------------------------------------------------
+
+|service| requires the following settings to create an Online Archive with
+the {+atlas-cli+}. You must specify these Online Archive settings either
+in the configuration file or as flags in the command:
+
+.. list-table:: 
+   :header-rows: 1 
+   :widths: 20 10 70 
+
+   * - Field 
+     - Type 
+     - Description 
+
+   * - ``groupId``
+     - string
+     - Unique identifier for your project. Your group id is the same 
+       as your project id. For existing groups, your group/project id 
+       remains the same. The resource and corresponding 
+       endpoints use the term groups.
+
+   * - ``archiveID``
+     - string
+     - Unique 24-hexadecimal digit string that identifies 
+       the online archive to update. This param is required for updating an
+       existing Online Archive.
+  
+   * - ``clusterName``
+     - string
+     - Human-readable label that identifies the cluster that contains 
+       the collection for which you want to create one online archive.
+
+   * - ``collName``
+     - string
+     - The name of your collection.
+
+   * - ``criteria``
+     - object
+     - Rules by which MongoDB MongoDB Cloud archives data.
+
+   * - ``dbName``
+     - string
+     - Human-readable label of the database that 
+       contains the collection that contains the online archive.
+
+For a full list of available settings, see the API documentation for 
+:oas-atlas-op:`Create One Online Archive<>`.
+
+Example Online Archive Create Command
+-------------------------------------
+After you create the file, run the command to create the Online Archive.
+
+.. code-block::
+
+    atlas clusters onlineArchive create --file online-archive-create.json --output json
```

Comment:
```suggestion
    atlas clusters onlineArchive create --file online-archive-config.json --output json
```
Suggestion: Lets rename the file to say `config` at the end instead of create -- I think we should omit the `create` in the file name since it's in the actual command, and may cause confusion to users.

Applies to the update section too.

Reply:
I think that it might be better to call it `online-archive-create-config.json` and `online-archive-update-config.json` because the configs are specific to those actions -- i.e. creating and updating an online archive respectively. WDYT?

Reply:
That's fair -- I like your suggestion!